lints/base.go

```diff
@@ -42,6 +42,17 @@ type LintInterface interface {
 	Execute(c *x509.Certificate) *LintResult
 }
 
+// An Enum to programmatically represent the source of a lint
+type LintType int
```

Comment:
This isn't a `Type` it's a Source. But we set `Source` to a more detailed string, so maybe this is a category? Alternatively, rename `Source` to `ReadableSource` and and then change this to source. @zakird, thoughts?

Reply:
We talked earlier today, "Type" was what we came up with. But I'm okay renaming `Source` to `ReadableSource`, and calling this `Source`.